### ISSUE TITLE
crazy thought

### ISSUE THREAD

**Issue by horak** [ecwalsh/lucidity#1]
ok so I've been thinking that it would be great to work on my writing skills...

I also think that some if not all of my dreams are relatively useless. I've thought of trying to use my dream recording as a way to work on my composition...

Hands down the most important part of dream journaling is working on recall, though it would be cool to see dream statistics.

Is there anyway to combine these two without losing one of the benefits? I was thinking that we somehow use our dream details as a writing prompt, or impose some writing prompt atop them, without construing the details in such a way to impair the statistics.

**Comment by ecwalsh** [comment]
There's potential here. Main thing for me, is that the statistics should be parsed rather than entered. 

I had a dream journal app that would ask me all sorts of questions -- if i went to the gym, if I drank coffee, if I drank water -- and it was overkill. At 6:30 a.m. I just want to type my fleeting memory down as fast as I can. If it becomes too much of a hassle, the entire thing is easy to abandon.

**Comment by horak** [comment]
yeah i can def see that. Maybe it could be an option on the day's that you're feeling inspired
